AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java

```diff
@@ -80,25 +80,25 @@
     public class ViewHolder extends RecyclerView.ViewHolder {
         public RelativeLayout deckLayout;
         public LinearLayout countsLayout;
-        public LinearLayout minIvlLayout;
+        public LinearLayout youngRevLayout;
         public ImageButton deckExpander;
         public ImageButton indentView;
         public TextView deckName;
         public TextView deckNew, deckLearn, deckRev;
-        public TextView minIvl;
+        public TextView youngRevCount;
 
         public ViewHolder(View v) {
             super(v);
             deckLayout = (RelativeLayout) v.findViewById(R.id.DeckPickerHoriz);
             countsLayout = (LinearLayout) v.findViewById(R.id.counts_layout);
-            minIvlLayout = (LinearLayout)v.findViewById(R.id.min_ivl_layout);
+            youngRevLayout = (LinearLayout) v.findViewById(R.id.min_ivl_layout);
```

Comment:
to maintain consistency, let's update the `android.id` field for this view in the xml file, `AnkiDroid/src/main/res/layout/deck_item.xml`, to be `young_rev_layout`

Android Studio should be able to perform the refactor for you. I think it's the same shortcut as on regular IntelliJ (Android Studio is built on top of IntelliJ, e hype). In case it is different/unfamiliar, move cursor to the name, hit `ctrl + t`, then select "rename." If `ctrl + t` isn't mapped to "refactor" then use `cmd + shift + a` and search "refactor" and you should be g2g.

Reply:
And also let's do the same for the other field.

If I can get to it today, I'm happy to knock out these changes myself (probably should have done that instead of writing a long comment...). But I'm tryna catch up on reviews, so unlikely that I will get to it immediately.

Reply:
Straight did not read the rest of the line to see that it still had the old name -- fixed now